README.md

```diff
@@ -90,6 +90,67 @@ let trifleSignedData = try trifle.createSignedData(
 let encodedTrifleSignedDataProto = try trifleSignedData.serialize()
 ```
 
+## Android SDK Usage 
```

Comment:
should the readme changes be a different PR?

Reply:
ehh - yes. But I slid it in :)